x-pack/plugins/lists/public/exceptions/components/builder/entry_renderer.test.tsx

```diff
@@ -1050,14 +1050,14 @@ describe('BuilderEntryItem', () => {
   test('it invokes "setWarningsExist" when invalid value in field value input', async () => {
     const mockSetWarningsExists = jest.fn();
 
-    (validateFilePathInput as jest.Mock).mockReturnValue('some warning message');
+    (validatePotentialWildcardInput as jest.Mock).mockReturnValue('some warning message');
     wrapper = mount(
       <BuilderEntryItem
         autocompleteService={autocompleteStartMock}
         entry={{
           correspondingKeywordField: undefined,
           entryIndex: 0,
-          field: getField('nestedField.nestedChild.doublyNestedChild'),
+          field: getField('file.path.text'),
```

Comment:
why are we changing this instead of adding a new test case?

Reply:
see note above